linux_sql/scripts/host_info.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+psql_host=$1
+psql_port=$2
+db_name=$3
+psql_user=$4
+psql_password=$5
+
+#check # of args
+if [ "$#" -ne 5 ]; then
+    echo "Illegal number of parameters"
+    exit 1
+fi
+
+#save machine statistics in MB and current machine hostname to variables
+vmstate_mb=$(vmstat --unit M)
```

Comment:
This variable name will work, but if we want to be consistent with the command, it should be vmstat (without the e - it stands for virtual memory statistics).